src/_includes/soft-mod-cheat-sheet-banner.html

```diff
@@ -2,7 +2,9 @@
 {% assign background_position = 'top left' %}
 {% capture title %}{% t software-modernisation.cheat-sheet-cta-banner-title %}{% endcapture %}
 {% capture description %}{% t software-modernisation.cheat-sheet-cta-banner-text %}{% endcapture %}
-{% capture cta_href %}{% t software-modernisation.cheat-sheet-cta-href %}{% endcapture %}
 {% capture cta_text %}{% t software-modernisation.cheat-sheet-cta-banner-link %}{% endcapture %}
```

Comment:
I think this is handled by the hubspot cta that gets inserted?

Reply:
Removed cta_text :)